### PR TITLE
Change epoch reward from 12.5K to 15K

### PR DESCRIPTION
**Pull request by zjshen14**
Implementation of https://github.com/iotexproject/iips/blob/master/iip-2.md

Also, cleanup some default rewarding settings in genesis.go, to make it consistent with the number we used in mainnet.

### CODE REVIEW

File: blockchain/genesis/genesis.go [hunk 1]
```diff
@@ -56,14 +56,15 @@ func defaultConfig() Genesis {
...
-			EpochRewardStr:                 unit.ConvertIotxToRau(300000).String(),
+			EpochRewardStr:                 unit.ConvertIotxToRau(12500).String(),
+			AleutianEpochRewardStr:         unit.ConvertIotxToRau(15000).String(),
 			NumDelegatesForEpochReward:     100,
 			ExemptAddrStrsFromEpochReward:  []string{},
-			FoundationBonusStr:             unit.ConvertIotxToRau(2880).String(),
+			FoundationBonusStr:             unit.ConvertIotxToRau(80).String(),
 			NumDelegatesForFoundationBonus: 36,
-			FoundationBonusLastEpoch:       365,
+			FoundationBonusLastEpoch:       8760,
```
Comment (dustinxie) [inline blockchain/genesis/genesis.go]: these change means we are changing epoch to 1 hour?

File: blockchain/blockchain.go [hunk 2]
```diff
@@ -611,6 +611,12 @@ func (bc *blockchain) MintNewBlock(
 			ActionGasLimit: bc.config.Genesis.ActionGasLimit,
 			Registry:       bc.registry,
 		})
+
+	if newblockHeight == bc.config.Genesis.AleutianBlockHeight {
```
Comment (dustinxie) [inline blockchain/blockchain.go]: not sure if I understand correctly, only the delegate who mint the aleutian block will perform the below update, others won't update new value 15K into admin state?

i think we need a special 'governance' type action to update 15K, and in commitBlock() everyone take that action to update to 15K? that seems a lot of change, but consider in the future we may run into similar change

or we should pass AleutianHeight to rewarding protocol, same as what we did for evm protocol?

File: blockchain/genesis/genesis.go (hunk 1 above)
Comment (zjshen14) [reply blockchain/genesis/genesis.go]: No, just make it equivalent to 1yr if epoch length is 1hr, make the default consistent with the mainnet setting

File: blockchain/blockchain.go (hunk 2 above)
Comment (zjshen14) [reply blockchain/blockchain.go]: > others won't update new value 15K into admin state?

Validator will update too

> that seems a lot of change, but consider in the future we may run into similar change

That's true, and it's not going to be a lot adjustment. Hence, it's not worthwhile yet of making an rewarding governance protocol

File: blockchain/blockchain.go [hunk 3]
```diff
@@ -1051,6 +1057,11 @@ func (bc *blockchain) runActions(
 			Registry:       bc.registry,
 		})
 
+	if acts.BlockHeight() == bc.config.Genesis.AleutianBlockHeight {
+		if err := bc.updateAleutianEpochRewardAmount(ctx, ws); err != nil {
```
[the remark this answers is not in the document]
Comment (zjshen14) [reply blockchain/blockchain.go]: Good call. Let's skip

File: blockchain/blockchain.go (hunk 2 above)
Comment (dustinxie) [reply blockchain/blockchain.go]: agreed, not worthwhile for now

**Pull request merged**